src/Illuminate/View/ViewServiceProvider.php

```diff
@@ -113,7 +113,7 @@ public function registerFactory()
 
 			$finder = $app['view.finder'];
 
-			$env = new Factory($resolver, $finder, $app['events']);
+			$env = new Factory($resolver, $finder, $app['events'], $app['cache']);
```

Comment:
We should be injecting the actual cache connection rather than the manger given that there's no way to use a specific driver later in your code.